README.rst

```diff
@@ -46,17 +56,26 @@ The following application notes use this library:
   * AN00181: xCORE-200 explorer accelerometer demo
 
 
-Required Software (dependencies)
-================================
+************
+Known Issues
+************
+  * The reg_ops_nack test fails on the XS1 architecture because it is unable to meet timing.
```

Comment:
I'd remove this given our level of support for XS1 these days (also the meta data above states for use with xcore-200/xcore.ai anyway)

Reply:
I think this new issue is worth referencing now: https://github.com/xmos/lib_i2c/issues/97

Reply:
I've removed the XS1 issue and referenced #97 instead